src/overlays/actors/ovl_En_Test7/z_en_test7.c

```diff
@@ -432,184 +419,188 @@ void EnTest7_Destroy(Actor* thisx, PlayState* play) {
     LightContext_RemoveLight(play, &play->lightCtx, this->lightNode);
 }
 
-void func_80AF19A8(EnTest7* this, PlayState* play) {
+void EnTest7_StartWarpCs(EnTest7* this, PlayState* play) {
     if (!CutsceneManager_IsNext(play->playerCsIds[PLAYER_CS_ID_SONG_WARP])) {
         CutsceneManager_Queue(play->playerCsIds[PLAYER_CS_ID_SONG_WARP]);
     } else {
         CutsceneManager_Start(play->playerCsIds[PLAYER_CS_ID_SONG_WARP], NULL);
-        func_80AF082C(this, func_80AF1A2C);
+        EnTest7_SetupAction(this, EnTest7_WarpCsPart1);
         play->unk_18844 = true;
     }
 }
 
-void func_80AF1A2C(EnTest7* this, PlayState* play) {
+void EnTest7_WarpCsPart1(EnTest7* this, PlayState* play) {
     Color_RGB8 fogColor = { 64, 0, 0 };
     Color_RGB8 ambientColor = { 220, 220, 255 };
-    f32 envLerp = this->unk_1E54 / 10.0f;
+    f32 envLerp = this->timer / (f32)(10 - 0);
```

Comment:
Why the `10 - 0`?

Reply:
If you look at the various lerps in this file, it's always from a ranger of times i.e. 

`envLerp = 1.0f - (sp2C / (f32)(110 - 100));` (from timer 110 to timer 100)
`f32 lerp = 1.0f - (temp / (f32)(100 - 96));` (from timer 100 to timer 96)
`lerp = (this->timer - 79) / (f32)(95 - 79);` (from timer 95 to timer 79)
`lerp = (this->timer - 42) / (f32)(68 - 42);` (from timer 68 to timer 42)
`lerp = (this->timer - 12) / (f32)(30 - 12);` (from timer 30 to timer 12)

and so on. So I just mimicked that style, even though one of the times is at timer 0